### PR TITLE
ANALYTICS-4490 | bookings_lead_type_added_to_lips_endpoint

### PR DESCRIPTION
**Pull request by martin-hristov-acc**
**References**: [ANALYTICS-4490](https://jira.gannett.com/browse/ANALYTICS-4490)

**Code PR**: https://github.com/GannettDigital/lead-ingestion-publication-service/pull/200/files

**Description**:
We need to add details for Booking events

### CODE REVIEW

File: source/includes/_lips_booking_events.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+Also referred to as "contact interactions", *events* are activities associated with a contact.  The event is a booking event.
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET `show` |events/:id|
+| PUT `update` | /events/:id |
+| PUT `update` | /events/:id/unarchive |
+| DELETE `destroy` | /events/:id |
```
Comment (brianroth-rl) [inline source/includes/_lips_booking_events.md:14]: isn't the route [booking_events](https://github.com/GannettDigital/lead-ingestion-publication-service/pull/200/files#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5R19)?

And I believe we should elaborate that this isn't the event id, but the booking_id.  If you want to interact with the booking event using the LIPS event id, this isn't the API for that.

File: source/includes/_lips_booking_events.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+### **BookingEvents**
+<a name="lips_booking_events"></a>
+## Booking Events API
+
+Also referred to as "contact interactions", *events* are activities associated with a contact.  The event is a booking event.
```
Comment (brianroth-rl) [inline source/includes/_lips_booking_events.md:5]: Ask product to provide something better to say here.

File: source/includes/_lips_booking_events.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+### Usage
+Use GET to retrieve events that match the query params.
+
+### GET (SHOW)
+Show a single event by id
+
+#### Examples:
+```
+
+curl -L -g -X GET 'https://data-connect-lips.ganettdigital.com/booking_events/1
```
Comment (brianroth-rl) [inline source/includes/_lips_booking_events.md:25]: The section above doesn't match the example (the example is correct)

File: source/includes/_lips_booking_events.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+| GET `show` |events/:id|
+| PUT `update` | /events/:id |
+| PUT `update` | /events/:id/unarchive |
+| DELETE `destroy` | /events/:id |
+
+### Usage
+Use GET to retrieve events that match the query params.
+
+### GET (SHOW)
+Show a single event by id
```
Comment (brianroth-rl) [inline source/includes/_lips_booking_events.md:20]: booking id, not just id.

**Review by brianroth-rl — CHANGES_REQUESTED** [review]

**Review by NBnfork — DISMISSED** [review]

File: source/includes/_lips_booking_events.md (hunk 2 above)
Comment (briannelson) [reply source/includes/_lips_booking_events.md:5]: Booking events happen when a Contact requests an Appointment through the appointments application.

File: source/includes/_lips_events.md [hunk 5]
```diff
@@ -146,6 +148,13 @@ curl -L -g -X GET 'https://data-connect-lips.ganettdigital.com/events?global_mas
 |call_recording_url| String | yes | Only included when interaction is call|
 |call_duration| Integer | yes | Length of call in seconds -- only included when interaction is call|
 
+**Booking**
+
+| Field Name | Datatype | Nullable | Description |
+|---|---|---|---|
+|booking_id| String | no | the external id of the booking event |
```
Comment (briannelson) [inline source/includes/_lips_events.md:155]: the external id (appointment token) of the Appointment. Used Internally to retrieve additional data about the Appointment.

**Review by Jordan3900 — APPROVED** [review]

**Pull request merged**